portfolio/src/main/webapp/projects.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Projects</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+      <div class = "content">
+          <h1> Past Projects </h1>
+          <nav>
+          <ul id = "horizontal-list">
+            <li><a href= "index.html">Home </a></li>
+            <li><a href="https://www.linkedin.com/in/ali-kew">LinkedIn</a></li>
+            <li><a href="https://github.com/alikew24"> Github </a></li>
+            <li><a href="projects.hmtl"> Projects </a></li>
+            <li><a href= "hobbies.html">Hobbies </a></li>
+          </ul>
+          </nav>
+          <h3> Connect Four </h3>
+          <img id = 'shape' src="images/connect-four-shorter-demo.gif"/>
+          <p> My senior year of highschool, I made a 
+              terminal based Connect Four game in Ruby. The user plays against a computer 
+              AI who tries to win! Check out my code on <a href="https://github.com/alikew24/ConnectFour"> Github</a>. </p>
+           <h3> Cornell Clubs and Organizations </h3>
+           <img id = 'shape' src="images/clubs.png"/>
```

Comment:
When the same `id` is used multiple times on a single page, you usually want a `class` instead.

Also, consider a more descriptive name like "project-image" ?